apps/src/p5lab/spritelab/SpritelabInput.jsx

```diff
@@ -6,10 +6,11 @@ import * as shapes from '../shapes';
 import {KeyCodes} from '@cdo/apps/constants';
 import {selectors} from '@cdo/apps/lib/tools/jsdebugger/redux';
 import {PromptType} from '../redux/spritelabInput';
-
+import {animationSourceUrl} from '../redux/animationList';
```

Comment:
Nit: Can we add a blank line here to separate the imports from the class? Makes it zone in when reading.

Reply:
updated! https://github.com/code-dot-org/code-dot-org/pull/45636/commits/de2e752831832b88479b4ca3eb55337de03ec365